providers/coordinator.go

```diff
@@ -103,13 +105,16 @@ func (c *coordinator) NewRuntime() *Runtime {
 }
 
 func (c *coordinator) newRuntime() *Runtime {
+	config.InitViperConfig()
```

Comment:
I think this shouldn't depend on the cli package. You would somehow need to pipe the `autoUpdate` value from outside the coordinator. I think the best would probably be to have the autoupdate settings defined on the coordinator itself since that setting is defined once per cnquery/cnspec instance. 
You would need to figure out from where to set the value though